### DISCUSSION TITLE
Comments on blog posts and subscription?

### DISCUSSION THREAD

**Discussion by mdandrews** [nmelhado/league-page#163] [General]
Hi thank you so much for the amazing tutorial and all your effort into creating this! A couple of things I’m struggling with.

How do I enable comments on my blog post?

Is there any way managers can subscribe to the blog so they get notified when a new post is published?

I have no coding experience and have just followed your training wheels guide so sorry if it’s a noob question!

Thanks so much for all the hard work!

Mark

**Comment by nmelhado** [comment]
Comments are enabled by default. However you can't see them or add them in the homepage (don't remember why I did that 😅)

As for subscription, I'm working on adding a separate tutorial for that...
